docs/faq.md

```diff
@@ -0,0 +1,67 @@
+# FAQ
+
+### What is HORNET?
+
+HORNET is a community driven IOTA fullnode. It is written in Go which makes it lightweight and fast.
+
+---
+
+### Does HORNET run on the Mainnet?
+
+Yes, HORNET gets actively tested within the IOTA Mainnet.
+
+---
+
+### Can I run HORNET on a Raspberry Pi?
+
+Yes, you can run HORNET on a Raspberry Pi 4B  with an external SSD. But we recommend to run HORNET on a more powerful device.
+
+---
+
+### I have difficulty setting up HORNET. Where can I get help?
+
+Our community loves helping you. Just ask your questions in the `#hornet` channel on the [official IOTA Discord Server](https://discord.iota.org/)
+
+---
+
+### What are the spammer settings for?
```

Comment:
These are the old settings for the old mainnet node.
The way to go should be to use the dashboard in the new node maybe.